packages/flutter_tools/gradle/new_settings.gradle

```diff
@@ -0,0 +1 @@
+include ':app'
```

Comment:
Can we have a better name than new_settings.gradle?

What is this supposed to be used for?

Reply:
this is indeed the "new" settings.gradle file. required after the migration is completed to AARs, 100%.

Reply:
Will it still be new six months from now?  Will this name make sense when we eventually adopt this path for everyone?

Reply:
This is an on-going effort, which will require to refactor some of the code later on. This file may be here for as long as we want to support the current version of settings.gradle. We currently don't have a version for Flutter Gradle plugin independently (Android does), which we should evaluate if that is the concern. If there was a version, then we can name this file accordingly.

Reply:
Talked offline - it seems like this would be settings_aar.gradle